src/test/resources/wiremock/stubs/users/find-system-user-by-query.json

```diff
@@ -3,8 +3,7 @@
     "method": "GET",
     "urlPath": "/users",
     "headers": {
-      "x-okapi-tenant": { "equalTo": "master" },
-      "x-okapi-token": { "equalTo": "X-Okapi-Token test value" }
+      "x-okapi-tenant": { "equalTo": "master" }
```

Comment:
why the okapi token was deleted from mocks?

Reply:
To not duplicate these stub files since they are shared with other system user tests